front/app/containers/Admin/index.tsx

```diff
@@ -116,14 +115,15 @@ const AdminPage = memo<Props & WithRouterProps>(
       pathname.includes('admin/initiatives') ||
       pathname.includes('admin/messaging') ||
       pathname.includes('admin/settings') ||
+      pathname.includes('admin/ideas') ||
       pathname.includes('admin/reporting');
 
     const fullWidth =
-      endsWith(pathname, 'admin/dashboard/moderation') ||
       pathname.includes('admin/dashboard') ||
       pathname.includes('admin/initiatives') ||
       pathname.includes('admin/messaging') ||
       pathname.includes('admin/settings') ||
+      pathname.includes('admin/ideas') ||
```

Comment:
On my screen (2k) it's not full width
![image](https://github.com/CitizenLabDotCo/citizenlab/assets/91883434/3c32ae74-3462-4ca0-a8b6-807f5c463fba)

But it's the same as on staging https://demo.stg.citizenlab.co/en/admin/ideas, so definitely not a blocker.